### PR TITLE
Opening frequencies analysis & export

### PR DESCRIPTION
**Pull request by amj**
Two scripts for the creation & export of information about openings.

opening_freqs.py -- pass it a directory & it reads all SGFs and extracts joseki information for each corner, writing it into a sqlite db.
opening_freqs_export.py -- pass it the sqlitedb created above, and it fills in a static html/js template (joseki.html) with the information about the most commonly played move.

Open the rendered html/js and a javascript framework (Vue) displays it and renders scatterplots.

Future improvements could make the export a server providing a dynamic backend to the go board & chart drawing bits.

### CODE REVIEW

File: oneoffs/opening_freqs_export.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
+"""TODO(jacksona): DO NOT SUBMIT without one-line documentation for opening_freq_export.
```
Comment (sethtroisi) [inline oneoffs/opening_freqs_export.py]: Remove or fix

Comment (amj) [reply oneoffs/opening_freqs_export.py]: Done, here and in other.

File: oneoffs/opening_freqs.py [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+                        (hr, c[1]))
+        db.commit()
+    db.close()
+
+
+def count_tenukis(seq):
+    colors = ''.join([c for c in seq if c in 'BW'])
+    # string.count only finds non-overlapping, and 'BWWW' is 2 tenukis
+    # thus this weird regex expr.
+    return (sum(1 for _ in re.finditer('(?=BB)', colors)) +
```
[the remark this answers is not in the document]
Comment (amj) [reply oneoffs/opening_freqs.py]: That is clever and i'll forget what it means when i look at this again in 6 months ;)

[the remark this answers is not in the document]
Comment (brilee) [reply oneoffs/opening_freqs.py]: `len(list(...))` seems a bit more straightforward here :P?

**Pull request merged**